A2rchi/interfaces/chat_app/app.py

```diff
@@ -197,3 +213,55 @@ def index(self):
     
     def terms(self):
         return render_template('terms.html')
+    
+    def like(self):
+        try:
+            # Get the JSON data from the request body
+            data = request.json
+
+            # Extract the HTML content and any other data you need
+            chat_content = data.get('content')
+            discussion_id = data.get('discussion_id')
+
+            feedback = {
+                "chat_content" :  chat_content,
```

Comment:
Will make a longer comment in PR; but I think we may want to include a `message_id` here, which is unique within a discussion.